assets/helpers/checkoutForm/checkoutForm.js

```diff
@@ -45,14 +48,33 @@ export const formInputs = (formClassName: string): Array<HTMLInputElement> => {
   return [];
 };
 
-export const formIsValid = (formClassName: string) => {
-  const form = document.querySelector(`.${formClassName}`);
-  if (form && form instanceof HTMLFormElement) {
-    return form.checkValidity();
+// Takes either a form element, or the HTML class of the form
+export const formIsValid = (form: Object | string) => {
+
+  let formElement;
+
+  if (typeof form === 'string') {
```

Comment:
might be nicer to have `formIsValid` only take a form element, and query for the element before calling it

Reply:
👍